package.json

```diff
@@ -53,5 +53,8 @@
     },
     "engines": {
         "node": ">=18.15.0"
+    },
+    "dependencies": {
```

Comment:
dependency should be added in `packages/components`

Reply:
I resolved this in the next commit. I moved the dependency to package.json in packages/components